frontend/src/views/RoomSelectView.vue

```diff
@@ -2,7 +2,22 @@
   <div :class="$style.container">
     <img :class="$style.logo" src="/logo.svg" alt="Logo" />
     <div :class="$style.header">部屋を選んで入室</div>
-    <div :class="$style.rooms">
+    
+    <div v-if="isLoading || (roomData.length === 0 && retryCount < maxRetries)" :class="$style.loading">
+      部屋一覧を読み込み中...
+    </div>
```

Comment:
```suggestion
    <div v-if="isLoading || (roomData.length === 0 && retryCount <= maxRetries)" :class="$style.loading">
      部屋一覧を読み込み中...（再試行 {{ retryCount }} / {{ maxRetries }}）
    </div>
```
他の場所で提案した変更 +α で、今何回目の再試行かがユーザーにも見えたら嬉しいかも知れません

Reply:
自分の好みなのですが、あえて意図してユーザー側に余計な情報見せてないです。
ごちゃごちゃするのあんまりかなーってのと、そんなに一回あたり試行時間かからない+3回繰り返すだけだから再試行の時間が短めなので。

ただ、気持ち変わって実装したくなった時のために、変数(`retryCount`, `maxRetries`)用意して簡単に足せるようにしてます！

(ちなみに、一応コンソールにはlog残してたはずです〜)